managed/src/main/java/com/yugabyte/yw/cloud/aws/AWSCloudImpl.java

```diff
@@ -33,6 +35,24 @@ public AmazonEC2 getEcC2Client(Provider provider, Region r) {
       .build();
   }
 
+  @Override
+  public boolean isAWSValidCredentials(Map<String, String> config) {
+    BasicAWSCredentials awsCreds = new BasicAWSCredentials(config.get("AWS_ACCESS_KEY_ID"),
```

Comment:
See how credentials are taken in `AWSCloudImpl.getEcC2Client()` (using AWSCredentialsProvider).

Reply:
Here we will have credentials before hand, we don't need to fallbackToDefault. 
BasicAWSCredentials serves the purpose here.

Reply:
OK, but anyway extract constants "AWS_ACCESS_KEY_ID" and "AWS_SECRET_ACCESS_KEY".

Reply:
I think @SergeyPotachev has a point here.
Shouldn't the purpose of preflight check be to ensure that AWSCloudImpl.getEcC2Client() will work?
i.e. it should actually use the exact same way of getting creds and then make sure it works. For example if user has not setup access key and selected IAM profile option. This preflight check will fail even though the actual code is going to work. 
What am I missing?

Reply:
i.e. should the method be more generic `isValidCreds` or even `canConnectToCloudProvider` as opposed to are these access keys valid. That leads to reusable and maintainable code where your preflight checks will use exact same code to create awsclient  that you are going to use for actual work during flight.

Reply:
Yeah! My bad, missed IAM things in that case, we should fallback to default like mentioned above. Thanks for this catch.